<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vvOverlayPanel</class>
 <widget class="QWidget" name="vvOverlayPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>323</width>
    <height>447</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="currentImageLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Image :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="vFFrame">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../vvIcons.qrc">:/new/prefix1/icons/cursor-uparrow.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="4">
       <widget class="QLabel" name="vectorFieldNameLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Deformation field :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Subsampling : </string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSpinBox" name="subSamplingSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string> Scale : </string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QSpinBox" name="scaleSpinBox">
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="5">
       <widget class="QLabel" name="coordinatesLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Displacement :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="5">
       <widget class="QLabel" name="normLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Length :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="5">
       <widget class="QCheckBox" name="lutCheckBox">
        <property name="text">
         <string>Use Logarithm LUT</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="compareFrame">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../vvIcons.qrc">:/new/prefix1/icons/GPSup.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLabel" name="imageComparedLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="colorLabel">
        <property name="text">
         <string>Color :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSlider" name="colorHorizontalSlider">
        <property name="maximum">
         <number>180</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="QLabel" name="refValueLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Pixel value in image 1 :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QLabel" name="valueLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Pixel value in image 2 :</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QLabel" name="diffValueLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Pixel difference :</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="fusionFrame">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="maximumSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../vvIcons.qrc">:/new/prefix1/icons/fusion.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="4">
       <widget class="QLabel" name="dataFusionnedLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Fusion :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="opacityLabel">
        <property name="text">
         <string>Opacity :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="3">
       <widget class="QSlider" name="opacityHorizontalSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Colormap :</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QComboBox" name="fusionColorMapComboBox">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>B&amp;W</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Heat</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cold</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Full Color Range</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Window :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QDoubleSpinBox" name="windowSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>-999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>10.000000000000000</double>
        </property>
        <property name="value">
         <double>100.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Level :</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QDoubleSpinBox" name="levelSpinBox">
        <property name="decimals">
         <number>1</number>
        </property>
        <property name="minimum">
         <double>-999999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>999999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>10.000000000000000</double>
        </property>
        <property name="value">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="5">
       <widget class="QLabel" name="valueFusionnedLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Pixel value in image 2 :</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../vvIcons.qrc"/>
  <include location="../vvIcons.qrc"/>
 </resources>
 <connections/>
</ui>
